packages/next/lib/typescript/runTypeCheck.ts

```diff
@@ -88,12 +88,20 @@ export async function runTypeCheck(
   const allDiagnostics = ts
     .getPreEmitDiagnostics(program as import('typescript').Program)
     .concat(result.diagnostics)
+
+  const diagnosticsWithoutIgnoredFiles = allDiagnostics
     .filter((d) => !(d.file && regexIgnoredFile.test(d.file.fileName)))
 
+  if (allDiagnostics.length !== diagnosticsWithoutIgnoredFiles.length) {
+    warn(
+      'Some test files reported TypeScript type errors, proceeding anyway.'
```

Comment:
I don't see the value of seeing this message, if those file are ignored why should take care of this? on top of this we dont show which files either, which causes more confusion.

Reply:
The point of this is to inform the user that errors are being ignored.

Currently a user of NextJS has no way of knowing that “Checking validity of types” does not check the validity of all types. It’s surprising to the user when running tsc produces different results than next build.

Solutions can come in many forms
- updating documentation
- Updating existing “Checking validity of types” logs
- Add a warning (what I chose)
- Stop ignoring type errors


The idea is that through a change like one of the above, developers will know that some type errors are being ignored. If you dislike the messaging, perhaps it would be more clear if I mentioned that the user can run tsc for more information.